hnn_core/gui/gui.py

```diff
@@ -1015,6 +1019,58 @@ def _get_evoked_widget(name, layout, style, location, data=None,
     return drive, drive_box
 
 
+def _get_tonic_widget(name, layout, style, data=None):
+
+    default_data = {
+        'amplitude': 0,
+        't0': 0,
+        'tstop': 0,
+        'seedcore': 14,
```

Comment:
```suggestion
```
is `seedcore` actually used anywhere? `net.add_tonic_drives` doesn't have any randomization so probably safe to delete

Reply:
Removed